other-cel/enforce-pod-duration/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: enforce-pod-duration
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: ../enforce-pod-duration.yaml
+    - patch:
+        resource:
+          apiVersion: kyverno.io/v1
+          kind: ClusterPolicy
+          metadata:
+            name: pod-lifetime
+          spec:
+            validationFailureAction: Enforce
+    - assert:
+        file: chainsaw-step-01-assert-1.yaml
```

Comment:
Reminder comment to rename `chainsaw-step-01-assert-1.yaml` files to `policy-ready.yaml` after converting policies in this PR

Reply:
I have renamed them